contract_manager/package.json

```diff
@@ -9,7 +9,9 @@
     "lib/**/*"
   ],
   "scripts": {
-    "build": "tsc",
+    "build": "pnpm run build:cjs && pnpm run build:esm",
+    "build:cjs": "tsc -p tsconfig.json",
+    "build:esm": "tsc -p tsconfig.esm.json",
```

Comment:
This should probably get reverted, this needs more testing before we should ship something like it